### PR TITLE
remove unneeded status element from json response on /categories

### LINKED ISSUE #15056: Ghost section in the apps page when app store is disabled
### Steps
1. Set "appstoreenabled" to false in config.php
2. Open the apps page
3. Click below "Not enabled"

ghostsection

Observed on master

### PR DESCRIPTION
**Pull request by DeepDiver1975**
fixes #15056

@karlitschek @PVince81 @nickvergessen please review - THX

### CODE REVIEW

**Comment by karlitschek** [comment]
nice :+1:

File: settings/controller/appsettingscontroller.php [hunk 1]
```diff
@@ -78,7 +78,7 @@ public function listCategories() {
 			}
 		}
 
-		$categories['status'] = 'success';
+//		$categories['status'] = 'success';
```
Comment (DeepDiver1975) [inline settings/controller/appsettingscontroller.php]: grrr - one should kill the line - not comment .....

**Comment by DeepDiver1975** [comment]
bäääh - github-fu

**Comment by PVince81** [comment]
:+1: fixes the issue

**Comment by PVince81** [comment]
Jenkins PR: https://github.com/owncloud/core/pull/15111

**Comment by PVince81** [comment]
@owncloud-bot retest this please

**Pull request merged**